doc/examples/jupyter_notebook/Makefile

```diff
@@ -1,12 +1,11 @@
 build:
 	docker build -t pachyderm_jupyter .
 
-prep: build
+prep:
 	pachctl version
-	./citibike_data/load.sh
-	./weather_data/load.sh
+	cd citibike_data && ./load.sh
+	cd weather_data && ./load.sh
 	pachctl create-pipeline -f ./sales_data/pipeline.json
-	pachctl create-job -f jupyter.json
```

Comment:
It sounds like there is a race here. By removing this and letting the user run the `make launch-jupyter` task we haven't fixed the race. It sounds like your working theory is that there was not any data available. Can we add a `flush-commit` command in here so that it blocks until the data is ready, then create the jupyter job?